src/OpalCompiler-Core/RBVariableNode.extension.st

```diff
@@ -12,7 +12,7 @@ RBVariableNode >> binding: aSemVar [
 
 { #category : #'*opalcompiler-core' }
 RBVariableNode >> isClean [
```

Comment:
On the nodes level #isClean sounds quite bad. It would be nice to find better name